test_kubeyaml.py

```diff
@@ -41,18 +52,10 @@ def strip(s):
                    strats.just('__') | \
                    strats.integers(min_value=1, max_value=5).map(lambda n: '-' * n)
 
-image_tags = strats.from_regex(r"^[a-z][\w.-]{0,127}$").map(strip)
+image_names = strats.lists(elements=image_components(),
```

Comment:
This gets reassigned not much later on -- does it need to be here, or need another name perhaps.